wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -5873,7 +5886,7 @@ func ClientSetCommand(ctx context.Context, pk *scpacket.FeCommandPacketType) (sc
 			return nil, fmt.Errorf("error updating client openai api token: %v", err)
```

Comment:
please change these messages as well.  should just be "ai api token" instead of "openai"